### PR TITLE
Feature/27285 dummy test fixtures

### PR DESCRIPTION
**Pull request by danchr**
https://redmine.magenta-aps.dk/27285

This PR switches the test fixture to a freshly generated fixture, makes generating said fixtures easier, and moves us slightly closer to the containerisation of our test infrastructure. These changes shave ~5 minutes off a test run.

- [ ] Have you updated the documentation?
- [ ] Have you performed a smoke test of the application?
- [ ] Have you written tests for your changes?
- [ ] Have you updated the release notes?

### CODE REVIEW

File: backend/mora/cli/base.py [hunk 1]
```diff
@@ -251,8 +251,16 @@ def expand_suite(suite):
...
+@click.option('--fixture', type=click.Choice(['minimal', 'simple', 'dummy',
+                                              'small', 'normal', 'large']),
+              default='dummy',
+              help='Choose the initial dataset.')
+@click.option('--dump-to', type=click.File(mode='w', lazy=True, atomic=True),
+              help='Dump the database to the given file.')
+@click.option('--backend-only', is_flag=True,
+              help="Don't run the ``vue-cli-service`` server for frontend "
+              "development.")
+def full_run(fixture, dump_to, backend_only):
```
Comment (cmoesgaard) [inline backend/mora/cli/base.py]: This function has a lot going on now. 150 lines of multiple if/else structures. I'd suggest a refactor into smaller, more manageable chunks.

File: backend/mora/cli/lora.py [hunk 2]
```diff
@@ -12,20 +12,16 @@
...
 import flask_saml_sso
-import pyexcel
 
 from . import base
 from .. import exceptions
 from .. import lora
-from .. import settings
 
 
 @base.cli.group('lora')
```
Comment (cmoesgaard) [inline backend/mora/cli/lora.py]: Do we use any of this anymore? If not, I suggest we just remove all of it.

File: backend/tests/fixtures/update-fixtures.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+do
+    echo
+    echo $fixture
+    echo
+
+    "$TOPDIR"/flask.sh full-run \
+             --fixture $fixture --dump-to "$CURDIR/$fixture.sql"
+done
+
+mv "$CURDIR/normal.sql" "$CURDIR/dummy.sql"
```
Comment (cmoesgaard) [inline backend/tests/fixtures/update-fixtures.sh]: Maybe we should just rename our use of `dummy.sql` to `normal.sql` instead? :slightly_smiling_face:

File: frontend/package.json [hunk 4]
```diff
@@ -23,6 +23,7 @@
     "lodash.reduce": "^4.6.0",
     "lodash.sortby": "^4.7.0",
     "moment": "^2.22.2",
+    "node-fetch": "^2.3.0",
```
Comment (cmoesgaard) [inline frontend/package.json]: Don't we already have a library for performing HTTP requests?

Comment (danchr) [reply frontend/package.json]: > Don't we already have a library for performing HTTP requests?

Right, `axios`… forgot about that one 🙂

**Pull request merged**